lib/services/ios-debug-service.ts

```diff
@@ -227,12 +236,9 @@ class IOSDebugService implements IDebugService {
         }
     }
 
-    private openDebuggingClient(): IFuture<void> {
+    private openDebuggingClient(fileDescriptor: string): IFuture<void> {
         return (() => {
-            this.$projectDataService.initialize(this.$projectData.projectDir);
-            let platformData = this.$platformsData.getPlatformData(this.platform);
-            let frameworkVersion = this.$projectDataService.getValue(platformData.frameworkPackageName).wait().version;
-
+            let frameworkVersion = this.getProjectFrameworkVersion().wait();            
             let inspectorPath = this.getInspectorPath(frameworkVersion).wait();
```

Comment:
This can be moved in the `else` body because it is not used in the legacy scenario when we open Safari.